etc/ansible/roles/network-runner/providers/dellemc.os9.os9/conf_access_port.yaml

```diff
@@ -1,10 +1,27 @@
 ---
+- name: "dellemc.os9: get running config for interface"
+  dellemc.os9.os9_command:
+    commands: "show running-config interface {{ port_name }}"
```

Comment:
What if you filtered the output right at the switch?

`show running-config interface {{ port_name }} | grep description`
e.g.

```
OCT4-SW-TORS#show running-config interface tengigabitethernet 1/22 | grep description
 description OCT4-2 2nd
 ```
 
 and then the output can be directly applied

Reply:
That includes a space at the beginning of description (it's indented that way on the output), so there would need to be a bit more filtering regardless. I also think it's safer to use regex because the behavior of how to execute grep might change across versions, but the config output shouldn't change.